spec/std/bool_spec.cr

```diff
@@ -41,6 +41,13 @@ describe "Bool" do
     it { false.to_s.should eq("false") }
   end
 
+  describe "<=>" do
```

Comment:
Same comment with moving it up near the other operators?

Reply:
done